fendermint/app/settings/src/lib.rs

```diff
@@ -83,28 +84,13 @@ pub struct DbSettings {
     pub state_hist_size: u64,
 }
 
-#[serde_as]
+/// Settings affecting how we deal with failures in trying to send transactions to the local CometBFT node.
+/// It is not expected to be unavailable, however we might get into race conditions about the nonce which
```

Comment:
Do you mind elaborating on this a bit. What you mean is that if you retry after some other transaction is broadcast to CometBFT we may get a race as the retry has a lower nonce than the second one which was (for some reason) accepted first?  
```suggestion
/// It is not expected to be unavailable, however we might get into race conditions about the nonce which
```

Reply:
Ahm :dizzy_face: Maybe...? :sweat_smile: 

What I meant was that:
* CometBFT is expected to be there (it's calling us after all), so I'm not trying any exponential backoff here to wait until it's available, which is why the settings are unsophisticated.
* The race conditions arise from the fact that the operator might be using the keys to sign other transactions, such as releasing funds to the parent network, which is practically the only failure scenario I could think of where it's worth retrying a broadcast immediately with a fresh nonce. The retries can get into trouble if we were to use simultaneous broadcasts from different places - but arguably if we know we are doing that then we should add an internal queue to the broadcaster.